src/data/markdown/docs/02 javascript api/08 k6-http/10-batch- requests -.md

```diff
@@ -20,35 +24,20 @@ When each request is specified as an array, the order of the arguments for each
 | 3        | body (optional)   | string / object / ArrayBuffer | The body of the request if relevant. Can be set to `null` if not applicable but you want to set the last `params` argument. |
 | 4        | params (optional) | object           | [Params](/javascript-api/k6-http/params) like auth, custom headers and tags.                                                |
 
+<Blockquote mod="warning">
+When a Request definition is provided as an Array, the order of items is relevant. Check the `Position` column for the correct order.
+</Blockquote>
+
+#### String
+
+If an item is passed as a `string` then it will be automatically parsed as a `GET` request.
```

Comment:
I don't understand this. The request definition says that method and URL parameters are required and must be strings.

What information does this section add?

Reply:
Ah, now I see in the new "Shorthand" example that it looks like the `method` parameter is not required, at least in arrays. I think we need to update the table, right?

Does this shortcut work on objects, too? "Item" makes me think it's only arrays.

Proposed changes:

> As a shortcut, you can pass only the `URL` in arrays. k6 will automatically use `GET` for the request method.

_or, if it can work in an object too_

> If you define only the URL parameter, k6 will automatically use `GET` for the request method.

Reply:
Oh, I think I see how I got confused.

If I understand correctly, there are really three ways to define batch requests:

- As an array of strings

  ```js
  http.batch(['http://test.k6.io', 'http://test.k6.io/pi.php']);
  ```
- As an array of arrays:
 
   ```js 
    const responses = http.batch([
      ['GET', 'https://test.k6.io', null, { tags: { ctype: 'html' } }],
      ['GET', 'https://test.k6.io/style.css', null, { tags: { ctype: 'css' } }],
      ['GET', 'https://test.k6.io/images/logo.png', null, { tags: { ctype: 'images' } }],
    ]);
    ```
 - As an array of objects
 
    ```js
      const responses = http.batch([req1, req2, req3]);
      ```
 
 Maybe we can mention this information at the beginning. I suppose objects and array batch definitions are simple enough that we can group them in one table, but string should have it's own section.